### ISSUE TITLE
Don't send speed or course until status = T

### ISSUE THREAD

**Issue by Hakansv** [douwefokkema/marpa#3]
Avoid transfer speed and course to OCPN while status Q until we change to T. It looks a bit strange. 
This is a method I've tried.
```
@@ -717,10 +717,10 @@ void ArpaTarget::PassARPAtoOCPN( OCPN_target_status status ) {
     if ( bearing < 0 ) bearing += 360;
     //LOG_INFO(wxT("BR24radar_pi: $$$ send dist = %f, bearing = %f"), dist, bearing);
     s_TargID = wxString::Format( wxT( "%2i" ), target_id );
-    s_speed = wxString::Format( wxT( "%4.2f" ), speed_kn );
-    s_course = wxString::Format( wxT( "%3.1f" ), logbook [0].course );
+    s_speed = wxString::Format( wxT( "%4.2f" ), status == Q ? 0.0 : speed_kn );
+    s_course = wxString::Format( wxT( "%3.1f" ), status == Q ? 0.0 : logbook [0].course );
     s_target_name = wxString::Format( wxT( "MARPA%2i" ), target_id ); 
```

**Comment by douwefokkema** [comment]
implemented in marpa3

**Comment by douwefokkema** [comment]
Implemented and tested in marpa3 efc38420150

**Comment by Hakansv** [comment]
Implemented - Thanks
